mm2src/coins/eth.rs

```diff
@@ -5186,7 +5161,7 @@ impl EthCoin {
                 Ok(PayForGasOption::Legacy(LegacyGasPrice { gas_price }))
             },
             SwapTxFeePolicy::Low | SwapTxFeePolicy::Medium | SwapTxFeePolicy::High => {
-                let fee_per_gas = coin.get_eip1559_gas_fee().await?;
+                let fee_per_gas = coin.get_eip1559_gas_fee(false).await?;
```

Comment:
And here we do not give an option for GUI for use_simple param.
Does it make sense to have such a param then?

Reply:
I only tried keeping the logic the same with the addition of force-using simple. This is cuz one might want to have only a simple estimator for the streaming part.

The part above should be some swap code which is better done using non-simple estimator for more correctness. we still fall back to simple if the configured non-simple streamer fails.

Reply:
Okay, 
actually I thought the way how providers are configured is not final yet and may be changed when QA team starts using them depending on the license etc